src/commands/userlookup.ts

```diff
@@ -0,0 +1,54 @@
+import { InteractionResponseType, InteractionResponseFlags } from 'discord-interactions';

```

Comment:
```suggestion
import { InteractionResponseType } from 'discord-interactions';
```

Just a small thing for this line, removing the unused import